cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go

```diff
@@ -17,110 +17,48 @@ limitations under the License.
 package fuzzer
 
 import (
-	"time"
-
 	"github.com/google/gofuzz"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-	kubeletconfigv1alpha1 "k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig/v1alpha1"
-	kubeproxyconfigv1alpha1 "k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1"
-	utilpointer "k8s.io/kubernetes/pkg/util/pointer"
+	newfuzzer "k8s.io/apimachinery/pkg/api/testing/fuzzer"
+	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
+
 )
 
 // Funcs returns the fuzzer functions for the kubeadm apis.
-func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
-	return []interface{}{
-		func(obj *kubeadm.MasterConfiguration, c fuzz.Continue) {
-			c.FuzzNoCustom(obj)
-			obj.KubernetesVersion = "v10"
-			obj.API.BindPort = 20
-			obj.TokenTTL = &metav1.Duration{Duration: 1 * time.Hour}
-			obj.API.AdvertiseAddress = "foo"
-			obj.Networking.ServiceSubnet = "foo"
-			obj.Networking.DNSDomain = "foo"
-			obj.AuthorizationModes = []string{"foo"}
-			obj.CertificatesDir = "foo"
-			obj.APIServerCertSANs = []string{"foo"}
-			obj.Token = "foo"
-			obj.Etcd.Image = "foo"
-			obj.Etcd.DataDir = "foo"
-			obj.ImageRepository = "foo"
-			obj.CIImageRepository = ""
-			obj.UnifiedControlPlaneImage = "foo"
-			obj.FeatureGates = map[string]bool{"foo": true}
-			obj.APIServerExtraArgs = map[string]string{"foo": "foo"}
-			obj.APIServerExtraVolumes = []kubeadm.HostPathMount{{
-				Name:      "foo",
-				HostPath:  "foo",
-				MountPath: "foo",
-			}}
-			obj.Etcd.ExtraArgs = map[string]string{"foo": "foo"}
-			obj.Etcd.SelfHosted = &kubeadm.SelfHostedEtcd{
-				CertificatesDir:    "/etc/kubernetes/pki/etcd",
-				ClusterServiceName: "etcd-cluster",
-				EtcdVersion:        "v0.1.0",
-				OperatorVersion:    "v0.1.0",
-			}
-			obj.KubeletConfiguration = kubeadm.KubeletConfiguration{
```

Comment:
Why remove all this fuzzing instead of just changing the signature?

Reply:
there is something wrong when i copy the code, i will fix it.